src/Operation/CreateCollection.php

```diff
@@ -112,9 +111,6 @@ final class CreateCollection implements Executable
      *
      *    This is not supported for servers versions < 5.0.
      *
-     *  * typeMap (array): Type map for BSON deserialization. This will only be
-     *    used for the returned command result document.
-     *
```

Comment:
The typeMap option can be removed since we don't return the command results.

Reply:
I agree with not needing a deprecation notice for removing typeMap, as we won't throw an error when the option is present, it just won't do anything.

I would point out however that a `typeMap` option here is a pretty good indication that the caller intends to use the result, so it could be a way to notify users of the changing return type. Since it doesn't catch all cases, I'm not sure it's warranted though.

Reply:
https://github.com/mongodb/mongo-php-library/pull/1473